guide/inheritance-wallet/landing-page.md

```diff
@@ -0,0 +1,208 @@
+---
+layout: guide
+title: Inheritance wallet
+description: An in-depth UX reference design for a multi-key bitcoin wallet with inheritance features designed for families.
+nav_order: 6
+has_children: true
+permalink: /guide/inheritance-wallet/
+main_classes: -no-top-padding
+image: https://bitcoin.design/assets/images/guide/inheritance-wallet/header.png
+image_base: /assets/images/guide/inheritance-wallet/
+images_creation-wallet-basics:
+    - file: 
+      alt: 
+      caption: 
+
+
+---
+
+<!--
+
+Editor's notes
+
+This page covers ....  
+
+Illustration sources
+
+https://www.figma.com/file/h5GP5v5dYfpXXfEUXf6nvC/Family-inheritance-wallet?type=design&node-id=5542%3A2119&mode=design&t=sBtcvrDzb8MPtWaK-1
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/inheritance-wallet/header.jpg"
+   retina = "assets/images/guide/inheritance-wallet/header@2x.jpg"
+   alt-text = "Two parents handing bitcoin to their two children."
+   width = 1600
```

Comment:
Width and height do not match the `header.jpg` file size. Either adjust the width and height here, or the image size. I'd suggest what I mentioned in a comment above, to have a separate preview and header file.